microovn/cmd/microovn/cluster_add.go

```diff
@@ -35,7 +35,7 @@ func (c *cmdClusterAdd) Run(cmd *cobra.Command, args []string) error {
 
 	token, err := m.NewJoinToken(args[0])
 	if err != nil {
-		return err
+		return fmt.Errorf("Unable to add MicroCluster, name %q is taken",args[0])
```

Comment:
Now the error is completely hidden, should we include it at the end of the string after a `: `?

Would the text improve if we add the word "server", i.e. "... add server to Micr ...". (Personally I would actually prefer "node" but I see the help text uses the word "server").

`gofmt -d .` also has opinions on this line. (We really should get the `make check-lint` and our CI to detect those for us.)